cfme/networks/provider/__init__.py

```diff
@@ -301,6 +301,9 @@ def all(self):
 
         return network_providers
 
+    def instantiate(self, prov_class, *args, **kwargs):
```

Comment:
Required:
Remove this; we already have `instantiate` method
https://github.com/ManageIQ/integration_tests/blob/master/cfme/networks/provider/__init__.py#L323